test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -233,22 +233,10 @@ public EndToEndTestsBase(TTestFixture fixture)
             // or to use the local file system the format should be: UseLocalFileSystem=true;Path={path}
             string apiHubConnectionString = Environment.GetEnvironmentVariable("AzureWebJobsDropBox");
 
-            if (string.IsNullOrEmpty(apiHubConnectionString))
-            {
-                throw new ApplicationException("Missing AzureWebJobsDropBox environment variable.");
-            }
+            await Fixture.CleanupApiHubTest(apiHubConnectionString);
```

Comment:
Since this is done on fixture creation, we shouldn't need this call and the one below in this function, right?